tests_fp/test_freezable_fp.py

```diff
@@ -0,0 +1,164 @@
+"""Testing for ElfPy types"""
+from __future__ import annotations
+
+import unittest
+from dataclasses import dataclass
+
+from elfpy.types import freezable
```

Comment:
is this testing what you want? I see there's only 1 freezable class, but the only part that explicitly uses FixedPoint is the QuantityFP object, which I don't see being used here.